pkg/dao/rpm.go

```diff
@@ -170,23 +171,26 @@ func (r rpmDaoImpl) Search(orgID string, request api.SearchRpmRequest, limit int
 	//      inner join repositories on repositories.uuid = repositories_rpms.repository_uuid
 	//      left join repository_configurations on repository_configurations.repository_uuid = repositories.uuid
 	// WHERE (repository_configurations.org_id = 'acme' OR repositories.public)
-	//       AND repositories.public
+	//       AND ( repositories.url in (...)
+	//             OR repository_configurations.uuid in (...)
+	//       )
 	//       AND rpms.name LIKE 'demo%'
 	// ORDER BY rpms.name, rpms.epoch DESC
 	// LIMIT 20;
 
 	// https://github.com/go-gorm/gorm/issues/5318
 	dataResponse := []api.SearchRpmResponse{}
-	orGroup := r.db.Where("repository_configurations.org_id = ?", orgID).Or("repositories.public")
-	db := r.db.
+	orGroupPublicOrPrivate := r.db.Where("repository_configurations.org_id = ?", orgID).Or("repositories.public")
+	groupUrlOrUuid := r.getGroupUrlOrUuid(urls, uuids)
```

Comment:
```suggestion
	orGroupUrlOrUuid := r.getGroupUrlOrUuid(urls, uuids)
```
just so it's consistent with `orGroupPublicOrPrivate`